tests/Aspire.MySqlConnector.Tests/AspireMySqlConnectorExtensionsTests.cs

```diff
@@ -37,7 +44,9 @@ public void ReadsFromConnectionStringsCorrectly(bool useKeyed)
             host.Services.GetRequiredKeyedService<MySqlDataSource>("mysql") :
             host.Services.GetRequiredService<MySqlDataSource>();
 
-        Assert.Equal(ConnectionString, dataSource.ConnectionString);
+        string actualConnectionString = dataSource.ConnectionString;
+        string expectedConnectionString = new MySqlConnectionStringBuilder(ConnectionString).ConnectionString;
+        Assert.Equal(expectedConnectionString, actualConnectionString);
```

Comment:
```suggestion
        Assert.Equal(NormalizedConnectionString, dataSource.ConnectionString);
```

With that new property, the assertion (here, and repeated below) could be simplified.